src/apps/lwaftr/binding_table.lua

```diff
@@ -195,11 +192,11 @@ end
 -- Iterate over the set of IPv4 addresses managed by a binding
 -- table. Invoke like:
 --
---   for ipv4_lo, ipv4_hi, psid_info in bt:iterate_psid_map() do ... end
+--   for entry in bt:iterate_psid_map() do ... end
 --
--- The IPv4 values are host-endianness uint32 values, and are an
--- inclusive range to which the psid_info applies.  The psid_info is a
--- psid_map_value_t pointer, which has psid_length and shift members.
+-- The IPv4 values are host-endianness uint32 values. The key is a
+-- softwire_key_t which has a "addr" and psid_info is a psid_map_value_t
+-- pointer, which has psid_length and shift members.
```

Comment:
I think the old comment still applies -- the psid map is a rangemap that will coalesce contiguous ranges of keys that have the same value.  However I think perhaps this function is no longer needed, as I think right now the only user of this function was to write out the psid-map yang list, and that is no more.  Consider removing the function and comment.